src/Store/FileExplorerStore.re

```diff
@@ -30,7 +30,7 @@ module Effects = {
 let nodeOffsetByPath = (tree, path) => {
   let rec loop = (node: FsTreeNode.t, path) =>
     switch (path) {
-    | [] => failwith("Well, this is awkward (ie. unreachable)")
+    | [] => `NotFound(0)
```

Comment:
I think this should be `` `Found(0)``. If the directory you open is expanded in the file explorer it looks like it'll hit this, because it'll try to keep looking. And it'll probably open it in `revealPath`, so it might always hit this when opening a directory.

Reply:
Ah, you're right! Nice catch.

I'm able to reproduce this on developer builds by running `:e a-directory-in-file-explorer`. So it wasn't actually a packaged-vs-developer-builds issue... it just happened that the packaged build started it in a different directory.

Reply:
And the focus works correctly now with `Found(0)` too

Reply:
Awesome! That was an easy fix. Clearly I just assumed that opening directories is a silly thing to do that's not worth bothering with. And you know what they say about assumptions... 🤦‍♂